scraper/src/mindtouch2zim/download.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+from typing import IO
+
+import requests
+import requests.structures
+import urllib3
+import zimscraperlib.__about__
+import zimscraperlib.constants
+from zimscraperlib.download import stream_file as stream_file_orig
+
+from mindtouch2zim.constants import CONTACT_INFO, NAME, VERSION
+
+
+def get_user_agent() -> str:
```

Comment:
Why is this a method and not a constant?

Reply:
Fixed, it is now a property of the new `Context`, computed dynamically from constants and other context (e.g. contact info)